### PR TITLE
feat: add `updateLegalEntityStatus` mutation

### PR DESCRIPTION
**Pull request by kiurchv**

### CODE REVIEW

File: __schema__/legalEntities.graphql [hunk 1]
```diff
@@ -96,19 +96,22 @@ type NhsVerifyLegalEntityPayload {
...
+  "Legal entity status"
+  status: LegalEntityStatus!
 }
 
 """
-Return type for `deactivateLegalEntity` mutation.
+Return type for `UpdateLegalEntityStatus` mutation.
 
 In order to deactivate legal entity user must have a scope `legal_entity:deactivate`.
+In order to suspend legal entity user must have a scope `legal_entity:update`.
```
Comment (ephor) [inline __schema__/legalEntities.graphql]:
```suggestion
In order to update legal entity user must have a scope `legal_entity:update`.
```

File: __schema__/legalEntities.graphql [hunk 2]
```diff
@@ -96,19 +96,22 @@ type NhsVerifyLegalEntityPayload {
 }
 
 """
-Input for `deactivateLegalEntity` mutation.
+Input for `updateLegalEntityStatus` mutation.
 """
-input DeactivateLegalEntityInput {
+input UpdateLegalEntityStatusInput {
   "Legal entity database unique identifier."
   id: UUID!
```
Comment (ephor) [inline __schema__/legalEntities.graphql]:
```suggestion
  databaseId: UUID!
```

Comment (kiurchv) [reply __schema__/legalEntities.graphql]: We should use global ID here, so UUID is the wrong type here

Comment (kiurchv) [inline __schema__/legalEntities.graphql]:
```suggestion
  id: ID!
```

File: __schema__/legalEntities.graphql [hunk 3]
```diff
@@ -96,19 +96,22 @@ type NhsVerifyLegalEntityPayload {
...
+  "Legal entity status"
+  status: LegalEntityStatus!
```
Comment (kiurchv) [inline __schema__/legalEntities.graphql]: @svetavedmed It seems that there is missing the `statusReason` field. Also, there isn't clear what type should be the such field

File: __schema__/legalEntities.graphql [hunk 4]
```diff
@@ -151,6 +133,55 @@ type NhsCommentLegalEntityPayload {
...
+  "Denotes that legal entity is active."
+  ACTIVE
+  "Denotes that legal entity is suspended and has limited rights for actions in ehealth system."
+  SUSPENDED
+}
+
+"""
+Return type for `UpdateLegalEntityStatus` mutation.
+
+In order to suspend legal entity user must have a scope `legal_entity:update`.
```
Comment (kiurchv) [inline __schema__/legalEntities.graphql]:
```suggestion
In order to update the legal entity status user must have a scope `legal_entity:update`.
```

**Pull request merged**